core/src/com/unciv/models/translations/Translations.kt

```diff
@@ -443,7 +468,9 @@ private fun String.translatePlaceholders(language: String, hideIcons: Boolean):
 private fun String.translateIndividualWord(language: String, hideIcons: Boolean): String {
     if (Stats.isStats(this)) return Stats.parse(this).toString()
 
-    val translation = UncivGame.Current.translations.getText(this, language, TranslationActiveModsCache.activeMods)
+    val translation = UncivGame.Current.translations.getText(
+        this, language, TranslationActiveModsCache.activeMods
+    ).replace(digitsRegex) { it.value.toLong().tr(language) }
```

Comment:
Looks good, I personally can't think of a problem except for maybe performance, which should hopefully not be a problem